Makefile

```diff
@@ -48,7 +48,7 @@ GOIMPORTS_VERSION                  := latest
 HELM_DOCS                          := $(TOOLS_DIR)/helm-docs
 HELM_DOCS_VERSION                  := v1.11.0
 KO                                 := $(TOOLS_DIR)/ko
-KO_VERSION                         := v0.12.0
+KO_VERSION                         := main
```

Comment:
You could also pin to the commit you want so this doesn't pick up every change.

Reply:
so true!